network/models.py

```diff
@@ -5,6 +5,15 @@
 class User(AbstractUser):
     pass
 
+    def get_following_list(self):
```

Comment:
why dont you use these in https://github.com/dhita-irma/cs50w-network/pull/2/files#diff-23386aa9223bd5e93909e1ec52be8856R77-R91